crates/astria-sequencer/src/config.rs

```diff
@@ -31,6 +31,13 @@ pub struct Config {
     pub metrics_http_listener_addr: String,
     /// Writes a human readable format to stdout instead of JSON formatted OTEL trace data.
     pub pretty_print: bool,
+    /// If the oracle is enabled. If true, the `oracle_grpc_addr` must be set.
+    /// Should be true for validator nodes and false for non-validator nodes.
+    pub oracle_enabled: bool,
```

Comment:
we got rid of `enable_mint` :p it would be disabled for non-validator nodes, yeah.

> Also also... do you think we should be more specific about the oracle name in case we add further ones in the future? As in, these could be renamed to slinky_xxx?

not sure we would add multiple sidecars at this point, but naming `slinky` might be more specific

Reply:
Let's make this consistent with our other services and name this
```suggestion
    pub no_oracle: bool,
```